src/dotnet/APIView/proposals/ProductCentricProposal.md

```diff
@@ -0,0 +1,306 @@
+
+* Parse the file to extract Product metadata (name, description, namespaces)
+* Create the Product
+* Register the namespaces as proposals (in proposed status)
+
+#### **When a TypeSpec Review is uploaded**
+
+If it has metadata:
+
+* Create a Product from `tspconfig.yaml`
+* Attach the TypeSpec review as the canonical spec for that Product
+
+#### **When a language Review is uploaded**
+
+APIView will automatically:
+
+* Look at `CrossLanguagePackageId`
+* Find the matching Product
+* Attach the Review to the Product
+* Register its namespaces as proposals (if not already approved)
+
+---
+
+## Updated Data Model
+
+### Product Model
+
+```tsp
+/** New top-level Product entity */
+model Product {
+  /** GUID identifier for the Product. */
+  Id: string;
+  /** Human-friendly name of the Product. Otherwise, defaults to the namespace. */
+  DisplayName?: string;
+  Description?: string;
+  /** Service-level contacts. */
+  Owners?: string[];
+  /** Convenience field for the latest approved (or proposed) namespace. */
+  Namespace: string;
+  NamespaceInfo: ProductNamespaceInfo;
+  /** Associated Review IDs. */
+  ReviewIds: string[];
+  ChangeHistory?: ProductChangeHistory[];
+  CreatedOn: utcDateTime;
+  LastUpdatedOn: utcDateTime;
+  IsDeleted: boolean;
+}
+
+/** Product-level change history */
+model ProductChangeHistory extends ChangeHistoryModel {
+  ChangeAction: ProductChangeAction;
+}
+
+/** Types of changes that can occur to a Product. */
+enum ProductChangeAction {
+  Created,
+  ...
+  Deleted,
+  UnDeleted
+}
+```
+
+### Namespace Proposal Updates
+
+```tsp
+/** Aggregated namespace information for a Product. */
+model ProductNamespaceInfo {
+  /** Currently approved namespaces. */
+  ApprovedNamespaces: NamespaceDecisionEntry[];
+  /** Full history of namespace proposals and decisions. */
+  NamespaceHistory: NamespaceDecisionEntry[];
+}
+
+/** Status of a specific namespace proposal under a Product. */
+enum NamespaceDecisionStatus {
+  Proposed,
+  Approved,
+  Rejected,
+  Withdrawn
+}
+
+/** A single namespace proposal (per language & package). */
+model NamespaceDecisionEntry {
+  Language: string;
+  PackageName: string;
+  Namespace: string;
+  Status: NamespaceDecisionStatus;
+  Notes?: string;
+  DecidedBy?: string;
+  DecidedOn?: utcDateTime;
+}
+```
+
+---
+
+## Review Model Changes
+
+Minimal updates to link Reviews to Products and remove namespace approval fields
+from Reviews.
+
+```diff
+ model Review {
+  ... existing fields ...
+-    NamespaceReviewStatus: NamespaceReviewStatus;
+-    NamespaceApprovalRequestedBy: string;
+-    NamespaceApprovalRequestedOn?: utcDateTime;
++    ProductId?: string;
++    IsTypeSpecReview?: boolean;
+ }
+```
+
+We will also remove the `NamespaceReviewStatus` enum.
+
+---
+
+## Key Workflows
+
+### Workflow 1 — Product Creation via TypeSpec Review
+
+1. EngSys parses `tspconfig.yaml` from a TypeSpec Review
```

Comment:
Is this part of TypeSpec engsys pipeline or is it something done by TypeSpec APIview parser?